11-getting-started-with-Jupyter.Rmd

```diff
@@ -0,0 +1,350 @@
+notebook tab menu, or use the `Shift + Enter` keyboard shortcut.
+
+```{r markdown-cell-not-run, echo = FALSE, fig.cap = "A Markdown cell in Jupyter that has not yet been rendered and can be edited.", fig.retina = 2, out.width="100%"}
+image_read("img/markdown-cell-not-run.png") %>% 
+  image_crop("3632x900")
+```
+
+```{r markdown-cell-run, echo = FALSE, fig.cap = "A Markdown cell in Jupyter that has been rendered and exhibits rich text formatting. ", fig.retina = 2, out.width="100%"}
+image_read("img/markdown-cell-run.png") %>% 
+  image_crop("3632x900")
+```
+
+### Creating new Markdown cells
+
+To create a new Markdown cell in Jupyter, click the `+` button in the Jupyter 
+notebook tab menu. By default, all new cells in Jupyter start as code cells, so the cell format needs to be changed to be recognized and rendered as a Markdown cell. To do this, click on the cell with your cursor to 
+ensure it is activated, and then click on the drop-down box next to the 
+**Restart the kernel, then re-run the whole notebook** button (▶▶) in the 
+Jupyter notebook tab menu and changing it from "**Code**" to "**Markdown**".
+
+```{r convert-to-markdown-cell, echo = FALSE, fig.cap = "New cells are by default code cells. To create Markdown cells, the cell format must be changed.", fig.retina = 2, out.width="100%"}
+image_read("img/convert-to-markdown-cell.png") %>% 
+  image_crop("3632x900")
+```
+
+## Saving your work
+
+As with any file you work on, it is critical to save your work often so you 
+don't lose your progress! Jupyter has an autosave feature, where open files are 
+saved periodically. The default for this is every two minutes. You can also 
+manually save a Jupyter notebook by selecting **Save Notebook** from the 
+File menu, by clicking the dick icon on the tab menu of the Jupyter notebook,
+or by using a keyboard shortcut (`Control + S` for Windows, or `Command + S` for
+Mac OS). 
+
+## Best practices for running a notebook
+
+### Best practices for executing code cells
+
+As you might know (or at least imagine) by now, Jupyter notebooks are great for
+interactively editing, writing and running R code - this is what they were 
+designed for. Consequently, Jupyter notebooks are flexible in regards to code 
+cell execution order. This flexibility means that code cells can be run in any 
+arbitrary order using the **Run** (▶) button. This flexibility has a downside 
+since it can lead to Jupyter notebooks whose code cannot be executed in a linear 
+order (from top to bottom of the notebook). A non-linear notebook is problematic 
+because a linear order is the conventional way code documents are run, and 
+others will have this expectation when running your notebook. Finally, if the 
+code is used in some automated process, it will need to run in a linear order, 
+from top to bottom of the notebook.
+
+Even with the best intentions, the flexibility of a Jupyter notebook can 
```

Comment:
I like how you added pictures here to help describe the scenario. I think we can help the explanation by making this paragraph more explicitly linked to the diagram. We could reference the pictures throughout the explanation rather than just mentioning it at the end of the paragraph.

Reply:
Done in bb66e7a